### PR TITLE
Added safer words, CA words, and ability to switch word lists

### PR DESCRIPTION
**Pull request by jpslav**
Adds a "safer" words list (the original list with potentially awkward words removed), as well as @ianchesal's safer California school words.  Also adds the ability to configurably switch between word lists.

Supersedes #2.

Thanks to @ianchesal for pushing for starting us down this path of safer words.

### CODE REVIEW

**Comment by jpslav** [comment]
@joemsak can you review?  @ianchesal your comments welcome too of course.

**Comment by ianchesal** [comment]
Hey! Super cool. Thanks for implementing the safer lists. :+1:

File: README.md [hunk 1]
```diff
@@ -18,20 +18,25 @@ Or install it yourself as:
...
 
     Babbler.configure do |config|
       # Use a different number of adjectives at the start of the
       # phrase, e.g. with a value of 2 you might get 'hard ancient prosecutor'
       config.num_adjectives = 1
+      # Choose a different word list. Options:
+      #   :original - the original word list in Babbler
+      #   :safer_words_1 - the original list with potentially awkward words removed
+      #   :california_words - words 2nd graders in California are expected to know
+      config.word_list = :original
```
Comment (joemsak) [inline README.md:39]: Should `:safer_words_1` be the default?

**Comment by joemsak** [comment]
:+1: Just had a comment about the default config option

**Comment by jpslav** [comment]
Ideally `safer_words_1` would probably be the default, but since the README advertises a permalink of sorts (Babbler.babble(42) always gives the same babble), I don't want to change that behavior within the same major version.  @joemsak after your pending babbler work we can re-evaluate if we want to make a major version change and either advertise that the `babble(42)` is not guaranteed to survive major version changes (or even any version changes).

**Comment by joemsak** [comment]
Gotcha

**Pull request merged**